components/library/http-parser/http-parser.p5m

```diff
@@ -0,0 +1,31 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Denys Rtveliashvili
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/include/http_parser.h
+link path=usr/lib/$(MACH64)/libhttp_parser.so \
+    target=/root/oi-userland/components/library/http-parser/build/prototype/$(MACH)//usr/lib/$(MACH64)/libhttp_parser.so.$(COMPONENT_VERSION)
+file path=usr/lib/$(MACH64)/libhttp_parser.so.$(COMPONENT_VERSION)
+link path=usr/lib/libhttp_parser.so \
+    target=/root/oi-userland/components/library/http-parser/build/prototype/$(MACH)//usr/lib/libhttp_parser.so.$(COMPONENT_VERSION)
```

Comment:
Wrong target. It should be something target=usr/lib/libhttp_parser.so.$(COMPONENT_VERSION)